templates/search/asset/common_asset_list.html.twig

```diff
@@ -0,0 +1,10 @@
+
+{% set header_params = call(asset_class ~ '::getHeaderParameters') %}
+
+<pre>{{ asset_class ~ '::getHeaderParameters' }}</pre>
+
+{{ call('Html::header', header_params) }}
+
+{{ call('Search::show', [asset_class]) }}
```

Comment:
You can call `Glpi\Search\SearchEngine::show()` directly. The old `Search` class doesn't serve much purpose anymore except backwards compatibility.

Reply:
Done 👍